docs/proposals/11-scheduling.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Reservation-Based Scheduling
+
+oep-number: 11
+
+creation-date: 2024-09-30
+
+status: draft
+
+authors:
+
+- "@lukasfrank"
+
+
+reviewers:
+
+- "@afritzler"
+- "@balpert89"
+
+---
+
+# OEP-11: Reservation-Based Scheduling
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Summary
+Scheduling resources is a central process to increase utilization of cloud infrastructure. It not only involves the process to find any feasible `pool` but rather, if possible, to find an optimal compute `pool`.
+
+
+## Motivation
+
+With `ironcore` a multi-level hierarchy can be built to design availability/failure zones. `Machine`s can be created on every layer, which are subsequently scheduled on the compute `Node`s  in the hierarchy below. A scheduler on every layer should assign a `Machine` on a `Pool` which can fulfill it. This proposal addresses the known limitations of the current implementation:
```

Comment:
Instead of compute `Node` I would refer here to compute `Pools` or `MachinePools`.

Reply:
Also what are the current limitations of the implementation. We should list them here and point out that this proposal wants to address those.